package.json

```diff
@@ -3,7 +3,8 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "node ./bin/www"
+    "start": "node ./bin/www",
+    "test": "jasmine-node spec/ --captureExceptions --forceexit"
```

Comment:
Cool. I have now implemented mocha, since I have used it once before. Good to know that it is superior to jasmine.